static/js/components/EducationForm.js

```diff
@@ -0,0 +1,285 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import IconButton from 'react-mdl/lib/IconButton';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import Switch from 'react-mdl/lib/Switch';
+import FABButton from 'react-mdl/lib/FABButton';
+import Icon from 'react-mdl/lib/Icon';
+import Dialog from 'material-ui/Dialog';
+import { Card } from 'react-mdl/lib/Card';
+import _ from 'lodash';
+import moment from 'moment';
+
+import ProfileFormFields from '../util/ProfileFormFields';
+import { generateNewEducation } from "../util/util";
+import { saveAndContinue } from '../util/profile_edit';
+import { HIGH_SCHOOL } from '../constants';
+
+class EducationForm extends ProfileFormFields {
```

Comment:
This code is all copied from an existing class, i.e. this was the constructor for `EducationTab` before this change. I was trying to get the behavior I wanted with as little refactoring as possible, which I why I didn't change anything here.

I suppose it would make sense to do some refactoring at the same time, but I didn't want to make the PR more complicated or harder to review. This applies also to your comment on line 53 below, I could refactor that method as well but I held off on doing so because I'm really just renaming an existing class and changing some of the JSX in `render()`.

Does that make sense? I'm down to refactor these method if we think it's important, just wanted to share my thought process.